resources/lib/clientinfo.py

```diff
@@ -39,3 +39,6 @@ def getDeviceId(self):
     def getVersion(self):
         version = xbmcaddon.Addon(id="plugin.video.embycon").getAddonInfo("version")
         return version
+
+    def getClient(self):
+        return 'Kodi EmbyCon'
```

Comment:
ok, let me confirm with Luke just to make sure, I remember some logic being applied to different clients and while the connection worked fine, things like icons and some custom features were fix/removed for some clients.